lib/appsignal/cli/demo.rb

```diff
@@ -55,7 +55,7 @@ def run(options = {})
             puts "It may take about a minute for the data to appear on https://appsignal.com/accounts"
           else
             puts "Error: Unable to start the AppSignal agent and send data to AppSignal.com"
-            puts "Please use `appsignal diagnose` to debug your configuration."
+            puts "Please use the diagnose command (https://docs.appsignal.com/ruby/command-line/diagnose.html) to debug your configuration."
```

Comment:
can't we actually show both the command and the link to the docs?

Reply:
I didn't add it to the Elixir integration, as the command for running the diagnose is different in development and in packaged releases. This is also the case for the Ruby integration, which explicitly requires the environment to be passed, and more likely than not requires a `bundle exec` in front of the command. Linking to the page seemed like q quick fix for that.

Reply:
Sure, but if you run the demo command in Elixir, you've already found the way to run an AppSignal command either through Mix or the release binary. If it's not clear in Ruby how to run the diagnose command the interface should be more clear.